tests/cases/fourslash/formattingOptionsChange.ts

```diff
@@ -17,6 +18,7 @@ runTest("InsertSpaceBeforeAndAfterBinaryOperators", "1 + 2 - 3", "1+2-3");
 runTest("InsertSpaceAfterKeywordsInControlFlowStatements", "if (true) { }", "if(true) { }");
 runTest("InsertSpaceAfterFunctionKeywordForAnonymousFunctions", "(function () { })", "(function() { })");
 runTest("InsertSpaceAfterOpeningAndBeforeClosingNonemptyParenthesis", "    ( 1 )", "    (1)");
+runTest( "InsertSpaceAfterOpeningAndBeforeClosingNonemptyBrackets", "[ 1 ]", "[1]" );

```

Comment:
@mhegazy Your request to add a test for brackets with only commas is interesting. With the current formatting options, `[,]` becomes `[, ]`, if "insert space after comma delimiter" is true. With this PR, the result would be: `[ , ]` .